src/javaharness/java/arcs/webimpl/WebHarnessModule.java

```diff
@@ -7,13 +7,15 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import javax.inject.Singleton;
 
 @Module
 public abstract class WebHarnessModule {
 
   @Binds
   public abstract ArcsEnvironment provideStandaloneWebArcsEnvironment(ShellApiBasedArcsEnvironment impl);
 
+  @Singleton
```

Comment:
I don't think this matters now, but I think it should be scoped to the Module, that is, the lifetime of this Map should be the lifetime of the Harness containing the ArcsEnvironment that uses it, but we can tweak this later as Dagger scopes can be confusing.

Reply:
Thanks, let's chat about it when you're back.